docs/basic/hello-rust.md

```diff
@@ -12,7 +12,7 @@ Seperti pada umumnya bahasa pemrograman, belajar membuat program pasti diawali d
 
 Pembuatan project baru di Rust bisa dilakukan dengan 2 cara, dengan `cargo` atau tanpanya. Disini kita akan skip bagian pembuatan project tanpa `cargo`, karena akan butuh effort lebih banyak nantinya dalam mengelola package dan manajemen build.
 
-> Di Rust istilah project lebih dikenal dengan **package**, dan pada ebook ini maknanya adalah sama
+> Di Rust istilah project lebih dikenal dengan **package**, dan pada ebook ini maknanya adalah sama.
```

Comment:
the changes are good. but I would like to make a note for this specific sentence.

there are a lot of blockquoted sentences in this ebook, and some of them have `.` at the end of the sentence some others don't. I'll try to standardize those later on